pyomo/util/components.py

```diff
@@ -30,8 +29,13 @@ def rename_components(model, component_list, prefix):
 
     Examples
     --------
-    >>> c_list = list(model.component_objects(ctype=Var, descend_into=True))
-    >>> rename_components(model, component_list=c_list, prefix='special_')
+    >>> model = pyo.ConcreteModel()
+    >>> model.x = pyo.Var()
+    >>> model.y = pyo.Var()
+    >>> c_list = list(model.component_objects(ctype=pyo.Var, descend_into=True))
+    >>> new = rename_components(model, component_list=c_list, prefix='special_')
+    >>> str(new)
+    "ComponentMap({'special_x (key=...)': 'x', 'special_y (key=...)': 'y'})"
```

Comment:
Shouldn't this be in a `doctest` block?

Reply:
It's not necessary.  Things that look like Python interactive sessions in the Examples are automatically picked up / tested.